ArcadiaParties-SPA/src/app/content/content.component.ts

```diff
@@ -31,9 +31,9 @@ export class ContentComponent implements OnDestroy {
 
     const currentUserDepartment$ = this.store
       .pipe(
-        select(selectUser),
+        select(selectUserDepartmentId),
         takeUntil(routeParams$),
-        map(user => user.department.id)
+        map(id => id),
```

Comment:
Could you think and answer - what do you want to achieve with mapping of id to id?